coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,24 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
+from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
-from coalib.parsing.Globbing import glob_escape
 
 
 def main(log_printer=None, section: Section=None):
     configure_logging()
+    return delete_orig(log_printer, section)
+
+
+def delete_orig(log_printer=None, section: Section=None):
+    if log_printer is not None:
+        logging.warning('Using log_printer is deprecated, '
+                        'use python inbuilt logging instead.')
 
     start_path = get_config_directory(section)
-    log_printer = (LogPrinter(ConsolePrinter()) if log_printer is None
```

Comment:
why not replace this with `log_printer = logging` ?

or `log_printer = logging if log_printer is None else log_printer` , if we actually want to do deprecation, rather than not supporting the old approach.